2021-10-05 19:02:06,564 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-10-05 19:02:06,565 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\LENOVO\AppData\Local\Programs\Python\Python38\lib\site-packages\instabot\bot\bot.py
2021-10-05 19:02:06,565 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-10-05 19:02:08,785 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-10-05 19:02:08,785 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-10-05 19:02:08,786 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-10-05 19:02:08,786 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2021-10-05 19:07:09,411 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-10-05 19:07:09,411 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-10-05 19:07:09,412 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-10-05 19:07:09,413 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 10 minutes.
